sdv/metadata/table.py

```diff
@@ -404,24 +404,25 @@ def _get_transformers(self, dtypes):
         for name, dtype in dtypes.items():
             field_metadata = self._fields_metadata.get(name, {})
             transformer_template = field_metadata.get('transformer')
-            if self._dtype_transformers[np.dtype(dtype).kind] is None:
-                transformers[name] = None
-
-            elif transformer_template is None:
+            if transformer_template is None:
                 transformer_template = self._dtype_transformers[np.dtype(dtype).kind]
```

Comment:
you can probably add this as the default to the get on line 406 so you only have to do this check once. ie
```python
transformer_template = field_metadata.get('transformer', self._dtype_transformers[np.dtype(dtype).kind])
```